dev-tools/cmd/dashboards/export_dashboards.go

```diff
@@ -162,11 +169,13 @@ func main() {
 			directory := path.Join(path.Dir(*ymlFile), "_meta/kibana/6/dashboard")
 			err := os.MkdirAll(directory, 0755)
 			if err != nil {
-				fmt.Printf("ERROR: fail to create directory %s: %v", directory, err)
+				fmt.Fprintf(os.Stderr, "ERROR: fail to create directory %s: %v", directory, err)
```

Comment:
This is missing a `\n` which cause some ugliness on the terminal.

Reply:
I replaced all calls by a function now that adds the `\n` at the end.